docs/resources/samples/combine-worksheets-into-single-workbook.md

```diff
@@ -81,27 +83,37 @@ interface WorksheetData {
 
 1. Sign into [Power Automate](https://flow.microsoft.com) and create a new **Instant cloud flow**.
 1. Choose **Manually trigger a flow** and select **Create**.
-1. Get all the files in the folder. In this example, we'll use a folder named "output". Add a **New step** that uses the **OneDrive for Business** connector and the **List files in folder** action. Provide the folder path that contains the .csv files.
-    * **Folder**: /output
+1. Add a **New step** to get all the workbooks you want to combine from their folder. Use the **OneDrive for Business** connector and the **List files in folder** action. For the **Folder** field, use the file picker to select the "output" folder.
```

Comment:
In this step, **New step** is capitalized, but in the next two steps it's all lowercase. For consistency, I recommend using the same capitalization for all three mentions (I have no strong opinion about which capitalization, though).